dvc/testing/benchmarks/cli/commands/test_repro.py

```diff
@@ -0,0 +1,25 @@
+def test_repro(bench_dvc, tmp_dir, dvc, dataset):
+    dvc.add(dataset)
```

Comment:
You should be able to use `make_dataset`, that will just copy things over to cache/workspace without wasting time dvc-adding from scratch.

Btw, you might be able to just use existing project here, similar to how we test exp show and stuff.

Reply:
I think I want to keep it artificial for now so it scales to the various datasets in dvc-bench and doesn't vary too much based on things like model training time.